.circleci/config.yml

```diff
@@ -254,20 +252,13 @@ jobs:
               "$CIRCLE_BUILD_URL" \
               > version.json
       - run:
-          name: Prepare DB
-          command: |
-            ./manage.py migrate
-            ./manage.py update_actions
-      - run:
-          name: Starting server
-          command: gunicorn normandy.wsgi:application --bind localhost:8000
-          background: true
-      - run:
-          name: Waiting for web server to be available
+          name: Build Docker image for test
           command: |
-            ./ci/wait-for-it.sh localhost:8000 \
-              --timeout=30 --strict \
-              -- echo "Done waiting. It should work now."
+            docker-compose build
+            docker-compose run app sh -c "/app/bin/wait-for-it.sh db:5432 -- python manage.py migrate"
+            docker-compose run app python manage.py update_actions
+            docker-compose run app python manage.py initial_data
```

Comment:
You should also be able to replace these with `make refresh` and it'll do all of those